example/bottle/server.py

```diff
@@ -0,0 +1,32 @@
+from bottle import route, run, app, static_file
+import bugsnag
+
+from bugsnag.wsgi.middleware import BugsnagMiddleware
+
+bugsnag.configure(
+    api_key='698ffb05632f0ffcad3ebd16383507fd',
+)
+
+@route('/')
+def index():
+    return static_file('index.html', root='.')
+
+@route('/crash')
+def crash():
+    """Deliberately raises an unhandled error and crash the app.
+    """
+    raise(Exception('SomethingBad'))
+
+@route('/handled')
+def handled():
+    try:
+        x = 1/0
+    except ZeroDivisionError:
+        bugsnag.notify(ZeroDivisionError('Bottle demo: To infinity... and beyond!'))
```

Comment:
Something seems off here. You catch the `ZeroDivisionError` but don't use it.

It seems like this code do one of the following:
1) have `except ZeroDivisionError as e:` on the previous line, then call `bugsnag.notify(e)`
2) Avoid the `try`+`except` and just call `bugsnag.notify(ZeroDivisionError(...))`

Reply:
I have just lifted the functions from the `flask` example, but I take your point. I've amended both to match the `plain` example.